openlibrary/plugins/upstream/account.py

```diff
@@ -920,9 +922,20 @@ def csv_body() -> Iterable[str]:
     return '\n'.join(csv_body())
 
 
+def escape_csv_field(raw_string: str) -> str:
+    """
+    Formats given CSV field string such that it conforms to definition outlined
+    in RFC #4180.
+
+    Note: We should probably use
+    https://docs.python.org/3/library/csv.html
+    """
+    escaped_string = raw_string.replace('"', '""')
+    return f'"{escaped_string}"'
+
+
```

Comment:
I think that this should be inside of the `export_books` class, at the same level as the new `get_work_from_id` method.